repl/src/main/scala/ammonite/repl/interp/Preprocessor.scala

```diff
@@ -143,7 +145,13 @@ object Preprocessor{
           "Don't know how to handle " + code
         )
       }
+    }
 
+    def script(stmts: Seq[Seq[String]], wrapperId: String): Res[Seq[Preprocessor.Output]]={
```

Comment:
Given how simple this method is, I don't think it belongs in `Preprocessor`. This 4 lines of code could easily live outside, and I'd put it in the `processScript` function